### DISCUSSION TITLE
X11 connection rejected because of wrong authentication

### DISCUSSION THREAD

**Discussion by abaracedo** [cypress-io/cypress#25622] [Docker]
Hi, there!

I'm upgrading from `4.0.1` to `12.4.0` and from `cypress/base:10` to `cypress/base:16.18.1`. 

When I run `cypress open <url>` I'm getting this error: `X11 connection rejected because of wrong authentication.` and I don't know why. Before upgrading the to the new image all was working fine and the GUI was opened.

My Dockerfile looks like:
```
FROM cypress/base:16.18.1

WORKDIR /app
ADD package.json .
ADD yarn.lock .

RUN apt-get update && apt-get install curl -y

RUN yarn install
RUN $(yarn bin)/cypress verify

ADD cypress.config.js /app
```

And the command I'm using to run through Docker is: 
```
docker run --rm -d -ipc=host --net=host
-e DISPLAY
-e CYPRESS_baseUrl={url}
-v cypress:app/cypress
-v /tmp/.X11-unix:/tmp/.X11-unix:rw
-v /etc/machine-id:/etc/machine-id
-v /home/ubuntu/.Xauthority:/root/.Xauthority
--entrypoint yarn image_name:version start
```

**Comment by Em2see** [comment]
Hello @abaracedo 
I think you need to check owner of **/home/ubuntu/.Xauthority**, because inside the container you use **/root/.Xauthority** under **root** user.

**Reply by abaracedo** [reply]
The thing is that the `docker` command works with the older image (cypress/base:10) it justs stopped working and throwing the auth error when changing it to `cypress/base:16.18.1`

**Comment by abaracedo** [comment]
Is it possible that something changed between the images, any dependency or setting?
